Ok we need some fixes

```
/Users/builder/azdo/_work/1/s/src/Core/src/ImageSources/iOS/ImageSourceExtensions.cs(21,49): error CS8604: Possible null reference argument for parameter 'attributes' in 'CGSize NSStringDrawing.GetSizeUsingAttributes(NSString This, UIStringAttributes attributes)'. [/Users/builder/azdo/_work/1/s/src/Core/src/Core.csproj::TargetFramework=net7.0-maccatalyst]
/Users/builder/azdo/_work/1/s/src/Core/src/Platform/iOS/LabelExtensions.cs(88,74): error CS8601: Possible null reference assignment. [/Users/builder/azdo/_work/1/s/src/Core/src/Core.csproj::TargetFramework=net7.0-maccatalyst]
/Users/builder/azdo/_work/1/s/src/Core/src/ImageSources/iOS/ImageSourceExtensions.cs(21,49): error CS8604: Possible null reference argument for parameter 'attributes' in 'CGSize NSStringDrawing.GetSizeUsingAttributes(NSString This, UIStringAttributes attributes)'. [/Users/builder/azdo/_work/1/s/src/Core/src/Core.csproj::TargetFramework=net7.0-ios]
/Users/builder/azdo/_work/1/s/src/Core/src/Platform/iOS/LabelExtensions.cs(88,74): error CS8601: Possible null reference assignment. [/Users/builder/azdo/_work/1/s/src/Core/src/Core.csproj::TargetFramework=net7.0-ios]
    36 Warning(s)
    4 Error(s)
```

--- a/eng/Version.Details.xml
+++ b/eng/Version.Details.xml
@@ -12,21 +12,21 @@
       <Uri>https://github.com/xamarin/xamarin-android</Uri>
       <Sha>dc07f98a55b1597c834bf7b5ebd6b10223244583</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.MacCatalyst.Sdk" Version="16.4.7042">
+    <Dependency Name="Microsoft.MacCatalyst.Sdk" Version="16.4.7044">
       <Uri>https://github.com/xamarin/xamarin-macios</Uri>
-      <Sha>07b6b0219b27c98eacc757b470ff6ee6dc5e5996</Sha>
+      <Sha>d657df2c33254f1470ddcc6d37e73bf51d22709c</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.macOS.Sdk" Version="13.3.7042">
+    <Dependency Name="Microsoft.macOS.Sdk" Version="13.3.7044">
       <Uri>https://github.com/xamarin/xamarin-macios</Uri>
-      <Sha>07b6b0219b27c98eacc757b470ff6ee6dc5e5996</Sha>
+      <Sha>d657df2c33254f1470ddcc6d37e73bf51d22709c</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.iOS.Sdk" Version="16.4.7042">
+    <Dependency Name="Microsoft.iOS.Sdk" Version="16.4.7044">
       <Uri>https://github.com/xamarin/xamarin-macios</Uri>
-      <Sha>07b6b0219b27c98eacc757b470ff6ee6dc5e5996</Sha>
+      <Sha>d657df2c33254f1470ddcc6d37e73bf51d22709c</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.tvOS.Sdk" Version="16.4.7042">
+    <Dependency Name="Microsoft.tvOS.Sdk" Version="16.4.7044">
       <Uri>https://github.com/xamarin/xamarin-macios</Uri>
-      <Sha>07b6b0219b27c98eacc757b470ff6ee6dc5e5996</Sha>
+      <Sha>d657df2c33254f1470ddcc6d37e73bf51d22709c</Sha>
     </Dependency>
     <Dependency Name="Microsoft.NET.Workload.Emscripten.net6.Manifest-7.0.100" Version="7.0.4" CoherentParentDependency="Microsoft.NETCore.App.Ref">
       <Uri>https://github.com/dotnet/emsdk</Uri>

--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -13,10 +13,10 @@
     <!-- xamarin/xamarin-android -->
     <MicrosoftAndroidSdkWindowsPackageVersion>33.0.52</MicrosoftAndroidSdkWindowsPackageVersion>
     <!-- xamarin/xamarin-macios -->
-    <MicrosoftMacCatalystSdkPackageVersion>16.4.7042</MicrosoftMacCatalystSdkPackageVersion>
-    <MicrosoftmacOSSdkPackageVersion>13.3.7042</MicrosoftmacOSSdkPackageVersion>
-    <MicrosoftiOSSdkPackageVersion>16.4.7042</MicrosoftiOSSdkPackageVersion>
-    <MicrosofttvOSSdkPackageVersion>16.4.7042</MicrosofttvOSSdkPackageVersion>
+    <MicrosoftMacCatalystSdkPackageVersion>16.4.7044</MicrosoftMacCatalystSdkPackageVersion>
+    <MicrosoftmacOSSdkPackageVersion>13.3.7044</MicrosoftmacOSSdkPackageVersion>
+    <MicrosoftiOSSdkPackageVersion>16.4.7044</MicrosoftiOSSdkPackageVersion>
+    <MicrosofttvOSSdkPackageVersion>16.4.7044</MicrosofttvOSSdkPackageVersion>
     <!-- Samsung/Tizen.NET -->
     <SamsungTizenSdkPackageVersion>7.0.105</SamsungTizenSdkPackageVersion>
     <!-- emsdk -->

--- a/src/Core/src/ImageSources/iOS/ImageSourceExtensions.cs
+++ b/src/Core/src/ImageSources/iOS/ImageSourceExtensions.cs
@@ -18,18 +18,22 @@ namespace Microsoft.Maui
 			var glyph = (NSString)imageSource.Glyph;
 
 			var attString = new NSAttributedString(glyph, font, color);
-			var imagesize = glyph.GetSizeUsingAttributes(attString.GetUIKitAttributes(0, out _));
+			var attributes = attString.GetUIKitAttributes(0, out _);
+			if (attributes != null)
+			{
+				var imagesize = glyph.GetSizeUsingAttributes(attributes);
 
-			UIGraphics.BeginImageContextWithOptions(imagesize, false, scale);
-			var ctx = new NSStringDrawingContext();
+				UIGraphics.BeginImageContextWithOptions(imagesize, false, scale);
+				var ctx = new NSStringDrawingContext();
 
-			var boundingRect = attString.GetBoundingRect(imagesize, 0, ctx);
-			attString.DrawString(new CGRect(
-				imagesize.Width / 2 - boundingRect.Size.Width / 2,
-				imagesize.Height / 2 - boundingRect.Size.Height / 2,
-				imagesize.Width,
-				imagesize.Height));
-
+				var boundingRect = attString.GetBoundingRect(imagesize, 0, ctx);
+				attString.DrawString(new CGRect(
+					imagesize.Width / 2 - boundingRect.Size.Width / 2,
+					imagesize.Height / 2 - boundingRect.Size.Height / 2,
+					imagesize.Width,
+					imagesize.Height));
+				
+			}
 			var image = UIGraphics.GetImageFromCurrentImageContext();
 			UIGraphics.EndImageContext();
 

--- a/src/Core/src/Platform/iOS/LabelExtensions.cs
+++ b/src/Core/src/Platform/iOS/LabelExtensions.cs
@@ -83,9 +83,8 @@ namespace Microsoft.Maui.Platform
 				StringEncoding = NSStringEncoding.UTF8
 			};
 
-			NSError? nsError = null;
-
-			platformLabel.AttributedText = new NSAttributedString(text, attr, ref nsError);
+			NSError nSError = new NSError();
+			platformLabel.AttributedText = new NSAttributedString(text, attr,ref nSError);
 		}
 
 		internal static void UpdateTextPlainText(this UILabel platformLabel, IText label)

--- a/src/Core/src/Platform/iOS/LabelExtensions.cs
+++ b/src/Core/src/Platform/iOS/LabelExtensions.cs
@@ -84,7 +84,8 @@ namespace Microsoft.Maui.Platform
 			};
 
 			NSError nSError = new NSError();
-			platformLabel.AttributedText = new NSAttributedString(text, attr,ref nSError);
+
+			platformLabel.AttributedText = new NSAttributedString(text, attr, ref nSError);
 		}
 
 		internal static void UpdateTextPlainText(this UILabel platformLabel, IText label)

